app/mailers/event/application_confirmation_mailer.rb

```diff
@@ -42,13 +42,13 @@ def placeholder_application_url
 
   def placeholder_missing_information
     missing = [nil, "", "nein", "non", "no"]
-    missing_questions = Event::Question.admin.joins(:answers)
+    missing_questions = safe_join(Event::Question.admin.joins(:answers)
       .where(answers: {participation: @participation, answer: missing})
       .pluck(:question)
-      .map { |question| ["<li>", question, "</li>"] }.flatten.join
+      .map { |question| [content_tag(:li, question)] }.flatten)
 
     return "" if missing_questions.blank?
 
-    "#{t("event.participations.missing_information")}<br><ul>#{missing_questions}</ul>"
+    t("event.participations.missing_information").html_safe + tag.br + content_tag(:ul, missing_questions)
```

Comment:
Enthält der übersetzte Text HTML? Wenn nicht, würde ich den auch nicht als html safe markieren.

Reply:
Das erste Objekt muss ein SafeBuffer sein, damit später beim escaping alles klappt, daher habe ich das so gemacht. Es wird die + Methode auf dem SafeBuffer aufgerufen und so ist das entstehende Objekt auf ein SafeBuffer wo dann aber der formatted teil unsafe ist. Alternativ kann man "".html_safe + schreiben, aber ich fand das weniger schön.

Reply:
Das dachte ich. In dem Fall würde ich aber `escape_html(t("event.....")) +` verwenden, da auch Übersetzungen nicht grundsätzlich vertrauenswürdig sind.